tests/integration/lxd/test_lxc.py

```diff
@@ -285,3 +286,16 @@ def test_info(instance, lxc, session_project):
     data = lxc.info(instance_name=instance, project=session_project)
 
     assert data["Name"] == instance
+
+
+@pytest.mark.lxd_instance
+def test_root_perms(instance: str, lxc: LXC, session_project: str) -> None:
+    proc = lxc.exec(
+        command=["stat", "-c", "%a", "/root"],
+        project=session_project,
+        instance_name=instance,
+        capture_output=True,
+        check=True,
+    )
+
+    assert int(proc.stdout) & 0o011
```

Comment:
These could be moved into `test_base.py` and use the [`session_provider`](https://github.com/canonical/craft-providers/blob/99d694cc69cdee035dd441eb68812a015123e9b4/tests/integration/conftest.py#L368-L384) fixture since they're really the same test :-)